app/controllers/search.js

```diff
@@ -43,15 +43,17 @@ export default Controller.extend({
       searchPhrase: this.inputPhrase,
       clicked: {
         type: 'article', // currently the only displayed type in the search
-        id: result.id,
+        id: result.wikiId + '_' + result.id,
```

Comment:
Won't this break existing tracking?

Reply:
`result` is the object from the model or from the view?

Reply:
https://wikia-inc.atlassian.net/browse/DE-4494 - that's the blocker I mentioned on daily - we agreed on this contract during meeting with DE last week.